### ISSUE TITLE
Setting a resolver is confusing

### ISSUE THREAD

**Issue by brantlymillegan** [ensdomains/ens-app#476]
When you try to set a resolver, it shows you a field that says "Enter Ethereum name or address." This is confusing because people could think they're supposed to put their own Ethereum address there. Also, 99.999% of people are going to use the Public Resolver, so it shouldn't be a small link under the field.

I would at least change the text to say something more like "Enter contract address of custom resolver or use Public Resolver".

And really, it should just default to the Public Resolver anyway, with setting a custom resolver being an "Advanced Option" or something.

**Comment by makoto** [comment]
Pending on https://github.com/ensdomains/ens-app/pull/511 to be merged (which is parked right now)

**Comment by makoto** [comment]
We could take Beltran's suggestion at https://github.com/ensdomains/ens-app/issues/345

**Comment by brantlymillegan** [comment]
@makoto Yes, simply changing the prompt text in the Resolver field would be an easy quick improvement, and then we can make bigger changes to the UI as we have more time for that

**Comment by makoto** [comment]
Merged and deployed https://github.com/ensdomains/ens-app/pull/525 . I will still keep this issue open until we merge PR 511

**Comment by virgil** [comment]
Eventually we'd want the to automatically filled-in with the public resolver and only under "advanced" settings would let you set your own resolver.

**Comment by jefflau** [comment]
@rliebert90

**Comment by makoto** [comment]
@jefflau @Arachnid @brantlymillegan @rliebert90  we now call `setSubnodeRecord` on creating subdomain which already sets the default resolver (and we even set eth record if `yourname.eth`) so it is no longer the issue

**Comment by brantlymillegan** [comment]
@makoto just tried to set a subdomain to test and the whole process failed
<img width="1792" alt="Screen Shot 2020-03-09 at 9 34 55 AM" src="https://user-images.githubusercontent.com/30162762/76236134-7798d380-61e9-11ea-9350-7f1d79c30f30.png">

In any event, this issue is mostly for setting the Public Resolver for a 2LD, which remains much more confusing than it needs to be. When people register a 2LD on .ETH, it should automatically set the resolver to the latest Public Resolver, as well as set the Address record to resolve to their Ethereum address that registered it. (People can always change the Resolver or Address later if they want to.)

**Comment by makoto** [comment]
@brantlymillegan your MM popup says you are pointing to 'localhost:8545'. Can you try with public testnet?

**Comment by brantlymillegan** [comment]
@makoto just tried on Infura Ethereum mainnet and then again with my local node and it worked this time. Great to see the subdomain automatically have the Public Resolver. Can we do the same for 2LDs, as well as set the Address record automatically to the Ethereum address that registered it?
